roles/sap_maintain_etc_hosts/tasks/update_host_absent.yml

```diff
@@ -26,7 +30,7 @@
   become_user: root
   become: true
 
-- name: Ensure that the entry all entries in hosts file are removed with name {{ thishost.node_name | d('undefined') }}
+- name: Ensure that all entries with name {{ thishost.node_name | d('undefined') }} in /etc/hosts are absent
```

Comment:
Here and other place did throw a lint error. According to lint variables have to be place a the ended of the name line. That is, why I had  this clumsy expression